l10n_de_tax_statement/views/report_tax_statement.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 BIG-Consulting GmbH (<https://www.openbig.org>)
+     Copyright 2019 Onestein (<http://www.onestein.eu>)
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <report
+        id="action_report_tax_statement"
+        model="l10n.de.tax.statement"
+        string="UstVA-Report"
+        report_type="qweb-pdf"
+        name="l10n_de_tax_statement.report_tax_statement"
+        file="l10n_de_tax_statement.report_tax_statement"
+    />
+    <record id="action_report_tax_statement" model="ir.actions.report">
+        <field name="paperformat_id" ref="paperformat_de_tax_statement" />
+    </record>
```

Comment:
To fix the following warning:

```
2021-08-25 08:45:20,912 14141 WARNING l10n_de_tax_statement_14_1 py.warnings: /home/onestein/Pycharm/Gitlab/odoo-14.0/src/odoo-14.0/odoo/tools/convert.py:296: DeprecationWarning: The <report> tag is deprecated, use a <record> tag for 'action_report_tax_statement'.
  warnings.warn(f"The <report> tag is deprecated, use a <record> tag for {xml_id!r}.", DeprecationWarning)
```
 
you should convert the `<report>` definition into the 'action_report_tax_statement', like it was done in https://github.com/onesteinbv/l10n-netherlands/commit/e7f9d7cabc32feb1d52e98b1a92c4012581ff613#diff-09d9700544e6db4c2464d41edce9f5577cab5fb596aacaa03ff547e9ea70b0ff

Reply:
You have now two definitions of `action_report_tax_statement`. Merge them.

Reply:
@hkapatel-initos I cannot see the print button
![Bildschirmfoto 2021-09-08 um 15 00 33](https://user-images.githubusercontent.com/8348598/132513964-6722f74d-c253-4d48-8690-e64aec3d3c73.png)